### DISCUSSION TITLE
BTP Requirement 1.1 Update v1.0.1. 20200409

### DISCUSSION THREAD

**Discussion by unamed12** [icon-project/btp#18] [General]
https://docs.google.com/document/d/1CgXx1y7vqidJDAnK4g0KQb-5qHSnH6GS/edit#heading=h.m3qicnmi52xp

Added requirements 
- ICON BMC
  - BMV Management 
     - Add a BMV
     - Delete a BMV
- PRA BMC
  - BSH Management
     - Request a BSH Registration
   - BMV Management
     - Add a BMV
     - Delete a BMV

**Comment by flowersinthesand** [comment]
We already covered BMV Management items but found that BSH Management item is changed:

- As is: An owner adds service.
- To be: Anyone (maybe) requests a service registration and an owner approves or rejects it.

I'll quickly discuss how to handle it and get back to you.

**Reply by flowersinthesand** [reply]
> To be: ~~Anyone (maybe)~~ requests a service registration and an owner approves or rejects it.

My bad. It should be that **BSH owner** requests a service registration and an owner approves or rejects it as the requirement defines.

We decided to accept this change in the development stage.
